README.md

```diff
@@ -1,18 +1,38 @@
 
-DAS Capture The Flag
-
-# das-apprentice-learning
+# DAS Capture The Flag
 Capture the Flag Game is a .NET Core 3.0 web application.  It is a multi-player browser game.
 Once two user's have connected and selected start the game will begin.
 
 The repository can be found here: https://github.com/SkillsFundingAgency/das-apprentice-learning
 
-# Database
+## Prerequisites
+
```

Comment:
You could fill this out, you have the information available to you, I can think of one pre-req off of the top of my head. If you're struggling to understand what goes here then have a look at some other projects readme's for some inspiration